includes/class-wc-taxjar-integration.php

```diff
@@ -477,18 +481,20 @@ public function create_or_update_tax_rate( $location, $rate, $tax_class = '', $f
 	}
 
 	public function smartcalcs_request( $json ) {
-		$url = $this->uri . 'taxes';
-
-		$this->_log( 'Requesting: ' . $this->uri . 'taxes - ' . $json );
-
-		$response = wp_remote_post( $url, array(
-			'headers' => array(
-							'Authorization' => 'Token token="' . $this->settings['api_token'] . '"',
-							'Content-Type' => 'application/json',
-						),
-			'user-agent' => $this->ua,
-			'body' => $json,
-		) );
+		$response = apply_filters( 'taxjar_smartcalcs_request', $json, false );
```

Comment:
@marcinbot This should return `false` rather than request params from `$json`:

```php
$response = apply_filters( 'taxjar_smartcalcs_request', false, $json );
```

Besides that, LGTM! 👍